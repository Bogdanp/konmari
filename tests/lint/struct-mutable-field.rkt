#lang racket/base

(require racket/string)

(provide
 user?
 user
 user-username
 set-user-username!
 user-password
 set-user-password!
 user-password-hash
 set-user-password-hash!)

(struct user
  (username [password-hash #:mutable])
  #:transparent)
