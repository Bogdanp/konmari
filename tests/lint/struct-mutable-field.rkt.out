/struct-mutable-field.rkt:9:2:error:identifier 'set-user-username!' provided but not defined
/struct-mutable-field.rkt:10:2:error:identifier 'user-password' provided but not defined
/struct-mutable-field.rkt:11:2:error:identifier 'set-user-password!' provided but not defined
